### PR TITLE
download thread refactorized waiting other threads like disk operations

### PR DESCRIPTION
**Pull request by alarraz**
try and wizard must run ok

### CODE REVIEW

File: src/engine/models/manager_hypervisors.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ def run(self):
                     logs.main.debug('Launching Download Changes Thread')
                     self.manager.t_downloads_changes = launch_thread_download_changes(self.manager)
 
-                    #launch brrom thread
+                    #launch brom thread
```
Comment (NefixEstrada) [inline src/engine/models/manager_hypervisors.py]:
```suggestion
                    #launch broom thread
```

File: dockers/app-devel/supervisord.conf [hunk 2]
```diff
@@ -15,21 +15,21 @@ stderr_logfile=/isard/logs/certs-error.log
...
-stdout_logfile=/isard/logs/webapp-supervisord.log
-stderr_logfile=/isard/logs/webapp-supervisord-error.log
+stdout_logfile=/isard/logs/webapp.log
+stderr_logfile=/isard/logs/webapp-error.log
 
 [program:engine]
 directory=/isard
-command=sh -c "sleep 15 && python3 run_engine.py 1>/isard/logs/engine.log 2>/isard/logs/engine-error.log"
-#command=python3 run_engine.py
+#command=sh -c "sleep 15 && python3 run_engine.py 1>/isard/logs/engine.log 2>/isard/logs/engine-error.log"
```
Comment (NefixEstrada) [inline dockers/app-devel/supervisord.conf]:
```suggestion
```

File: dockers/app-devel/Dockerfile [hunk 3]
```diff
@@ -1,15 +1,16 @@
 FROM isard/alpine-pandas:latest
 MAINTAINER isard <info@isard.com>
 
-RUN apk add --no-cache git yarn py3-libvirt py3-paramiko py3-lxml py3-xmltodict py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-flask-login py3-netaddr py3-requests curl
+RUN apk add --no-cache bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
```
Comment (NefixEstrada) [inline dockers/app-devel/Dockerfile]:
```suggestion
RUN apk add --no-cache git bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
```

File: dockers/app-devel/Dockerfile [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
+RUN apk add --no-cache git
```
Comment (NefixEstrada) [inline dockers/app-devel/Dockerfile]:
```suggestion
```

File: dockers/app-devel/Dockerfile (hunk 3 above)
Comment (jvinolas) [reply dockers/app-devel/Dockerfile]: git is not required anymore. Bash can be useful for debugging reasons, but I think it could be removed also.

Comment (NefixEstrada) [inline dockers/app-devel/Dockerfile]:
```suggestion
RUN apk add --no-cache git bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
```

File: dockers/app-devel/Dockerfile (hunk 4 above)
Comment (NefixEstrada) [inline dockers/app-devel/Dockerfile]:
```suggestion
```

File: dockers/app-devel/supervisord.conf (hunk 2 above)
Comment (NefixEstrada) [inline dockers/app-devel/supervisord.conf]:
```suggestion
```

File: src/engine/models/manager_hypervisors.py (hunk 1 above)
Comment (NefixEstrada) [inline src/engine/models/manager_hypervisors.py]:
```suggestion
                    #launch broom thread
```

**Pull request merged**